recipe/build.sh

```diff
@@ -5,12 +5,20 @@ set -ex
 # necessary to ensure the gobject-introspection-1.0 pkg-config file gets found
 # meson needs this to determine where the g-ir-scanner script is located
 export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$BUILD_PREFIX/lib/pkgconfig
+export PKG_CONFIG=$BUILD_PREFIX/bin/pkg-config
+
+if [[ "$CONDA_BUILD_CROSS_COMPILATION" == "1" ]]; then
+  MESON_ARGS="${MESON_ARGS} -Dintrospection=disabled"
```

Comment:
Is this desirable?

Reply:
No, far from it, but it doesn't matter much as it doesn't work for Windows either. This is also why we are still stuck with that old pango version: newer releases require the Harfbuzz introspection files to build the pango counterparts.